extensions/hibernate-reactive/runtime/src/main/java/io/quarkus/hibernate/reactive/runtime/FastBootHibernateReactivePersistenceProvider.java

```diff
@@ -359,10 +359,17 @@ private void registerVertxAndPool(String persistenceUnitName,
 
     private static void injectRuntimeConfiguration(HibernateOrmRuntimeConfigPersistenceUnit persistenceUnitConfig,
             Builder runtimeSettingsBuilder) {
+
+        String generationStrategy = persistenceUnitConfig.schemaManagement().strategy();
+        if (!"none".equals(generationStrategy) && persistenceUnitConfig.database().startOffline()) {
+            throw new PersistenceException(
+                    "Schema management strategy `quarkus.hibernate-orm.schema-management.strategy` can only be set to `none` when using offline mode `quarkus.hibernate-orm.database.start-offline=true`");
```

Comment:
Just a reminder that this duplicates the message from Hibernate ORM, so if you follow my suggestion of changing it there, you will need to change it here, too.